docs/docs/02200-tables/00400-access-permissions.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Table Access Permissions
+slug: /tables/access-permissions
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Access Permissions
+
+SpacetimeDB enforces different levels of table access depending on the context. All contexts access tables through `ctx.db`, but the available operations differ based on whether the context is read-write or read-only.
+
+## Reducers - Read-Write Access
```

Comment:
We should either amend this or add a section for read-write access from procedure `with_tx` transactions. I'll come back later and leave a suggestion, or push a commit, or something.